{
  "id": "CVE-2016-9015",
  "definition_id": "oval:org.opensuse.security:def:20169015",
  "title": "CVE-2016-9015",
  "description": "\n    Versions 1.17 and 1.18 of the Python urllib3 library suffer from a vulnerability that can cause them, in certain configurations, to not correctly validate TLS certificates. This places users of the library with those configurations at risk of man-in-the-middle and information leakage attacks. This vulnerability affects users using versions 1.17 and 1.18 of the urllib3 library, who are using the optional PyOpenSSL support for TLS instead of the regular standard library TLS backend, and who are using OpenSSL 1.1.0 via PyOpenSSL. This is an extremely uncommon configuration, so the security impact of this vulnerability is low.\n    ",
  "cvss3": "3.7/CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:L/I:N/A:N",
  "severity": "Low",
  "impact": "low",
  "affected": {
    "family": "unix",
    "platforms": [
      "openSUSE Tumbleweed"
    ],
    "cpes": [
      "cpe:/o:opensuse:tumbleweed"
    ]
  },
  "packages": [
    {
      "name": "python3-urllib3",
      "status": "released",
      "fixed_version": "0:1.18.1-1.2",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "python36-urllib3",
      "status": "released",
      "fixed_version": "0:1.26.6-1.2",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "python38-urllib3",
      "status": "released",
      "fixed_version": "0:1.26.6-1.2",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "python39-urllib3",
      "status": "released",
      "fixed_version": "0:1.26.6-1.2",
      "platform": "openSUSE Tumbleweed"
    }
  ],
  "references": [
    {
      "id": "Mitre CVE-2016-9015",
      "source": "CVE",
      "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-9015"
    },
    {
      "id": "SUSE CVE-2016-9015",
      "source": "SUSE CVE",
      "url": "https://www.suse.com/security/cve/CVE-2016-9015"
    },
    {
      "id": "SUSE-SU-2019:0139-1",
      "source": "SUSE-SU",
      "url": "https://lists.suse.com/pipermail/sle-security-updates/2019-January/005056.html"
    },
    {
      "id": "openSUSE-SU-2019:0159-1",
      "source": "SUSE-SU",
      "url": "https://lists.opensuse.org/opensuse-security-announce/2019-02/msg00014.html"
    },
    {
      "id": "unknown",
      "source": "SUSE-SU",
      "url": "https://lists.suse.com/archive/suse-security-announce/2007-Jun/0001.html"
    },
    {
      "id": "SUSE bug 1023502",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1023502"
    },
    {
      "id": "SUSE bug 1024540",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1024540"
    }
  ],
  "issued": "2021-10-02T00:00:00Z",
  "updated": "2022-02-17T00:00:00Z"
}
